src/main/java/it/gov/pagopa/payhub/activities/dto/classifications/TransferSemanticKeyWithOutComeCodeDTO.java

```diff
@@ -0,0 +1,14 @@
+package it.gov.pagopa.payhub.activities.dto.classifications;
+
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+import lombok.experimental.SuperBuilder;
+
+@Data
+@SuperBuilder
+@NoArgsConstructor
+@AllArgsConstructor
+public class TransferSemanticKeyWithOutComeCodeDTO extends TransferSemanticKeyDTO {
+	private String outcomeCode;
```

Comment:
Let's use the same name in the PaymentsReporting class
```suggestion
	private String paymentOutcomeCode;
```